src/main/java/com/easypost/app/WebhookExample.java

```diff
@@ -9,10 +9,10 @@
 public class WebhookExample {
 
     public static void main(final String[] args) {
-        EasyPost.apiKey = "cueqNZUb3ldeWTNX7MU3Mel8UXtaAMUi";
+        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
```

Comment:
Do we want to set these up to pull from the environment? They're just examples, why not just put a fake `secret_api_key` dummy text?

Reply:
Best practice to pull in creds not from directly in a script. Sensitive info like this should never be hard coded anyway.